app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsViewModel.kt

```diff
@@ -169,6 +171,7 @@ class GroupConversationDetailsViewModel @Inject constructor(
                         isServicesAllowed = groupDetails.conversation.isServicesAllowed(),
                         isUpdatingAllowed = isSelfAnAdmin,
                         isUpdatingGuestAllowed = isSelfAnAdmin && isSelfInOwnerTeam,
+                        mlsEnabled = isMLSEnabled(),
```

Comment:
We are now adding the `mlsEnabled` parameter to group options state but we are still not using it in any UI 🤔